build.yaml

```diff
@@ -101,13 +108,12 @@ common_cli:
     # choose to configured.
     # The list of ASCII codes it uses can be found here.
     # https://wiki.archlinux.org/index.php/Color_Bash_Prompt
-    parse_logs() {
+    function parse_logs {
       # ubuntu defaults to mawk, which causes some things to break
```

Comment:
:exclamation: I think rather than trying to work around various differences in `/bin/sh` we should actually make sure we are running in bash.

Or better, consider using a language other than shell that is available everywhere (python for example).